documentation/book/assembly-kafka-mirror-maker-whitelist.adoc

```diff
@@ -0,0 +1,19 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-mirror-maker-whitelist-{context}']
+
+= Whitelist
+
+It is possible to specify what are the topics that the Kafka Mirror Maker has to mirror from the source to the target Kafka cluster.
+The so called "whitelist" is defined in the `KafkaMirrorMaker` resource.
+It is a regular expression used for specifying from the simpler case with a single topic name to complex patterns.
+For example, it is possible to mirror two topics named A and B using "A|B" or all topics using "*".
+Multiple regular expressions separated by commas can be passed to Kafka Mirror Maker as well.
```

Comment:
This comma separation isn't documented on the CRD

Reply:
I opened the #917 for that.